docs/en/stack/ml/anomaly-detection/job-tips.asciidoc

```diff
@@ -98,4 +103,15 @@ If you are using {ece} or the hosted Elasticsearch Service on Elastic Cloud,
 `xpack.ml.max_model_memory_limit` is set to prevent you from creating jobs 
 that cannot be allocated to any {ml} nodes in the cluster. If you find that you 
 cannot increase `model_memory_limit` for your {ml} jobs, the solution is to 
-increase the size of the {ml} nodes in your cluster.
\ No newline at end of file
+increase the size of the {ml} nodes in your cluster.
+
+
+[[dedicated-indices]]
+===== Dedicated indices
+
+For each {anomaly-job}, you can optionally specify a separated index to store 
+the {anomaly-detect} results. As {anomaly-jobs} may produce a large amount 
+of results (for example, jobs with small bucket span or with long running 
```

Comment:
I think we might want to list many time series as an example here as it will probably be the most frequent reason for a job to produce too many results.